### PR TITLE
[SD-3920] Add a parser for scoop

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: superdesk/io/feeding_services/file_service.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def _update(self, provider):
                             with open(file_path, 'rt') as f:
                                 xml = ElementTree.parse(f)
                                 parser = self.get_feed_parser(provider, xml.getroot())
-                                item = parser.parse(xml, provider)
+                                item = parser.parse(xml.getroot(), provider)
```
Comment (sivakuna-aap) [inline superdesk/io/feeding_services/file_service.py]: Won't this affect other xml parsers?

Comment (marwoodandrew) [inline superdesk/io/feeding_services/file_service.py]: We don't currently have any file/xml ingests configured. I have tested a Reuters with a file feeding service and it requires this fix, haven't tested nitf source.

**Comment by petrjasek** [comment]
:+1:

File: superdesk/io/feed_parsers/scoop_newsml_2_0.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    NAME = 'scoop_newsml2'
+
+    def can_parse(self, xml):
+        if xml.tag.endswith('newsMessage'):
+            try:
+                self.root = xml
+                header = xml.find(self.qname('Header'))
+                origin = header.find(self.qname('origin')).text
+                if origin == 'BusinessDesk':
```
Comment (mugurrus) [inline superdesk/io/feed_parsers/scoop_newsml_2_0.py]: Why hardcode origin?

**Comment by mugurrus** [comment]
:+1:  (except minor comment)

File: superdesk/io/feed_parsers/scoop_newsml_2_0.py (hunk 2 above)
Comment (marwoodandrew) [inline superdesk/io/feed_parsers/scoop_newsml_2_0.py]: The parser is specifically for this source, I doubt any other provider would implement the same "quirks" in the standards.

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**